### PR TITLE
Add `version()` API to unify with Keras and KerasNLP

### PR DESCRIPTION
**Pull request by grasskin**
New `keras_cv.version()` API to keep parity with Keras and KerasNLP.

### CODE REVIEW

**Review by divyashreepathihalli — APPROVED** [review]

**Review by mattdangerw — APPROVED** [review]
Thanks!

**Comment by mattdangerw** [comment]
We should probably update to reflect the changes here...

https://github.com/keras-team/keras-nlp/pull/1332

Current version will break `python pip_build.py` I think.

File: keras_cv/version_utils.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from keras_cv.api_export import keras_cv_export
+
+# Unique source of truth for the version number.
+__version__ = "0.7.0"
```
Comment (mattdangerw) [inline keras_cv/version_utils.py:18]: Keep this a 0.8.0 (the upcoming release)

**Review by mattdangerw — APPROVED** [review]
lgtm! one comment

**Pull request merged**